src/Components/Navbar.css

```diff
@@ -1,3 +1,41 @@
 .p-menubar-button {
   display: none !important;
 }
+
+.p-menubar-root-list {
+  display: flex;
+  justify-content: center;
+}
+
+.p-menubar {
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.p-menubar-end {
+  margin-left: 12rem !important;
+}
+
+@media screen and (max-width: 550px) {
+  .p-menubar-end {
+    margin-left: auto !important;
+  }
+}
+
+@media screen and (max-width: 360px) {
+  .p-menubar-start {
+    width: 60% !important;
+  }
+}
+
+.linkfree-icon {
+  background-image: url('../images/linkfree.ico');
+  width: 2rem;
+  height: 2rem;
+  background-size: 2rem 2rem;
+}
+
+.p-menuitem-text {
+  font-weight: 600;
+  font-size: 20px;
+}
```

Comment:
I think it might look better if the icon and title disappeared on screens smaller than 864px, that's where the chips jump to 2 columns instead of 3.

Reply:
As mentioned in the comment below, this is handled by the primereact library to hide for screens smaller than 960px. We may need to override many properties to achieve this.